Automattic/Simplecons
# Add File Icon

## Fix

Add a new file icon (16px) for use in Electron import notes dialog. See below for illustration.

<img width="106" alt="Screenshot 2021-03-16 at 16 54 21" src="https://user-images.githubusercontent.com/36532468/111348605-4367b680-8678-11eb-8f33-059f75667299.png">

## To Test
- Verify the icon appears as intended on a 16px grid.
- Verify the SVG code is consistent with other icons.
